client/helpers/client/searchClient.js

```diff
@@ -62,6 +62,7 @@ Template.searchClient.helpers(
           const query = {};
           if (dataObject.isHMISClient) {
             query.query = `isHMISClient=true&schema=${dataObject.schema}`;
+            Router.go('preliminarySurvey', { _id: dataObject._id }, query);
```

Comment:
@PoojaGandhi123 Why this line is added ?

We don't have any route names `preliminarySurvey`